iocore/net/SSLNetVConnection.cc

```diff
@@ -621,7 +621,7 @@ SSLNetVConnection::net_read_io(NetHandler *nh, EThread *lthread)
           Debug("ssl", "ssl handshake for vc %p, expired, release the connection", this);
           read.triggered = 0;
           nh->read_ready_list.remove(this);
-          readSignalError(nh, VC_EVENT_EOS);
+          readSignalError(nh, EPIPE);
```

Comment:
I don't think EOS is correct, but EPIPE doesn't seem appropriate as well. I'd use ETIMEDOUT if we can change it to any error code and want to choose one from standard ESOMETHING error codes.

Reply:
You are correct.  Will fix this up.